<launch>

    <!-- 启动相机节点  -->
    <!-- <include file="$(find astra_camera)/launch/astra.launch" /> -->
    <include file="$(find usb_cam)/launch/usb_cam-test.launch"/>

    <!-- 启动笑脸识别节点  -->
    <node pkg="castlex_vision_detector" name="astra_smile" type="astra_smile.py" output="screen">
        <remap from="input_rgb_image" to="/usb_cam/image_raw" />      
        <rosparam>
            haar_scaleFactor: 1.2
            haar_minNeighbors: 2
            haar_minSize: 40
            haar_maxSize: 60
        </rosparam>
        <param name="faceCascade" value="$(find castlex_vision_detector)/data/haar_detectors/lbpcascade_frontalface.xml" />
        <param name="smileCascade" value="$(find castlex_vision_detector)/data/haar_detectors/haarcascade_smile.xml" />
    </node>

    <!-- 启动订阅cv_bridge_image话题  -->
    <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" args="/cv_bridge_image"/>

</launch>
